panel/models/reactive_esm.ts

```diff
@@ -570,6 +574,7 @@ export class ReactiveESM extends HTMLBox {
       // @ts-ignore
       this.compiled_module = importShim(url)
       const mod = await this.compiled_module
+      console.log(mod)
```

Comment:
Did you leave this on purpose?

Reply:
Nope, will add a linter rule to error on `console.log`.

Reply:
https://github.com/holoviz/panel/pull/7162